CHANGELOG.asciidoc

```diff
@@ -3,6 +3,238 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.12.0]]
+=== Beats version 7.12.0
+https://github.com/elastic/beats/compare/v7.11.2...v7.12.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Fix panic with inline SSL when the certificate or key were small than 256 bytes. {pull}23820[23820]
+
+*Auditbeat*
+
+*Filebeat*
+
+- Rename `s3` input to `aws-s3` input. {pull}23469[23469]
+
+*Heartbeat*
+- Refactor synthetics configuration to new syntax. {pull}23467[23467]
```

Comment:
@andrewvc can you confirm this is a breaking change?

Reply:
Yes, it is a breaking change, however, it's in an experimental feature